src/datanode/src/service.rs

```diff
@@ -95,6 +96,7 @@ impl<'a> DatanodeServiceBuilder<'a> {
         let config = GrpcServerConfig {
             max_recv_message_size: opts.rpc_max_recv_message_size.as_bytes() as usize,
             max_send_message_size: opts.rpc_max_send_message_size.as_bytes() as usize,
+            tls: TlsOption::default(),
```

Comment:
nit: There is some tech debt between `FrontendOptions` and `DatanodeOptions`. It's better to unify by using `GrpcOptions`. Maybe it can be refactored in the next PR.